src/components/form/_form.scss

```diff
@@ -41,15 +41,15 @@
     box-shadow: 0 2px 0px 0px $support-01;
 
     ~ .bx--form-requirement {
-      max-height: rem(200px);
+      opacity: 1;
```

Comment:
Not sure if this solution will work, as this will cause inconsistent heights when using multiple form items. The text will have a height, even when hidden using this method.